internal/pkg/otel/samples/windows/hostmetrics.yml

```diff
@@ -0,0 +1,28 @@
+receivers:
```

Comment:
Since we haven't included the `hostmetricsreceiver` for Windows, how about we remove the `windows` directory alltogether?